dipy/io/stateful_tractogram.py

```diff
@@ -36,6 +37,12 @@ class Space(enum.Enum):
     RASMM = 'rasmm'
 
 
+class Origin(enum.Enum):
+    """ Enum to simplify future change to convention """
+    NIFTI = False
+    TRACKVIS = True
+
```

Comment:
`True`, `False`, What does it mean? Why there are not just integers? Does it imply we can not add any other origin type?

Reply:
Well the Enum was there just in case someone forget which standard is which.
I could remove it altogether and continue with a direct use of true and false. But it seemed a bit more explicit to use this instead of boolean
But there is indeed only two type (which is why they it was a boolean value in the first place)

Reply:
This enum is misleading @frheault. Please suggest alternative direction.

Reply:
Well I could put `NIFTI = 'nifti'` and `TRACKVIS = 'trackvis'` or `NIFTI = 'center'` and `TRACKVIS = 'corner'`

But either way the goal remains the same, to have a value to check simply to switch between one state to the other. Which one do you prefer ?

Reply:
Well I could put NIFTI = 'nifti' and TRACKVIS = 'trackvis' or NIFTI = 'center' and TRACKVIS = 'corner'

But either way the goal remains the same, to have a value to check simply to switch between one state to the other. Which one do you prefer ?

Reply:
I understand your point @frheault, but I think `Enum` is not the correct structure for this boolean case. I try to find an alternative to propose to you.

Reply:
I can remove it all together. The function can accept either `'center'` or `'corner'`. This was there just in case someone forgot which one was which one.

People can just read the docstring and see which are the acceptable option. I can also write a proper logging error so people know the two options if they fail to provide an acceptable string

Reply:
I mean, `Enum` is ok, it is just the way you are using it. Just try to avoid the boolean `True` or `False`. it should work without this boolean.

Reply:
@skoudoro what about using `NIFTI = 'center'` and `TRACKVIS = 'corner` ?

Would not that be what you are saying ?

Reply:
Let's go for that, they are much more meaningful